docs/testing.rst

```diff
@@ -0,0 +1,32 @@
+.. _testing:
+
+======================
+Tests for Page Objects
+======================
+
+``web-poet`` provides :ref:`tools for testing Page Objects
+<web-poet:web-poet-testing>`. ``scrapy-poet`` projects can use a Scrapy command
+to easily generate tests::
+
+    scrapy create_po_test my_project.po.MyItemPage http://example.com/page1.html
+
+This will request the provided page, create an instance of the provided Page
+Object for this page, request its :meth:`~web_poet.pages.ItemPage.to_item`
+method and save both the Page Object dependencies and the resulting item as a
+test fixture. These fixtures can then be used with the ``pytest`` plugin
+provided by ``web-poet``.
+
+Configuring the test location
+=============================
+
+The ``SCRAPY_POET_TESTS_DIR`` setting specifies where to create the tests. It
+can be set in the project settings or with the ``-s`` command argument.
+
+Handling time fields
+====================
```

Comment:
Should we also document this feature in web-poet docs? Or maybe also remove the explanation from here and only say that we set this value?